base-helm-configs/barbican/barbican-helm-overrides.yaml

```diff
@@ -122,13 +122,13 @@ pod:
       api:
         min_available: 0
   resources:
-    enabled: false
+    enabled: true
     api:
       requests:
         memory: "128Mi"
         cpu: "100m"
       limits:
-        memory: "1024Mi"
+        memory: "4096Mi"
```

Comment:
Do we know when using these upper limits that we will have enough memory to support full pod utilization and horizontal scaling?

Reply:
some rough math means we will be consuming 210G of ram if every pod on a single control plane was running at 100% max memory utilization

Reply:
DFW.stage three controllers all have 128G of ram.  SJC 6 controllers all have 512G of ram.